hw/dv/data/common_sim_cfg.hjson

```diff
@@ -41,11 +41,11 @@
   tl_dbw: 4
 
   // Default UVM verbosity settings
-  n: UVM_NONE
-  l: UVM_LOW
-  m: UVM_MEDIUM
-  h: UVM_HIGH
-  d: UVM_DEBUG
+  expand_uvm_verbosity_n: UVM_NONE
```

Comment:
Can we drop the 'uvm_' on these? It is just a sim cfg detail that can stay out of the Python code.

Reply:
That's kind of the point, though: the Python code is in `SimCfg.py`, which runs simulations. The rename is to ensure that stuff to do with UVM simulations (which expands to names containing "UVM_") is tagged. Or have I misunderstood you?

Reply:
The Python code in `SimCfg.py` builds the sim object from the HJson regardless of whether the testbench is UVM-enabled (example: all of our IPs) or not (example: PRESENT, GALOIS LFSR).

The point was to be able to translate generically named verbosity levels {n, l, m, h, d} set on the command line into simulation technology specific ones {UVM_NONE, ...}. Let's say some old testbench is using OVM - the same command line args should translate to {OVM_NONE, ...}. Hence, the Python code must be free from such assumptions. 

An alternative is to change the generically names ones to {NONE, LOW, ...}, and get rid of these expansions completely. We just update the `run_opts` to 
```
  run_opts: [..., "+UVM_VERBOSITY=UVM_{verbosity}", ...]
```
The downside of this is `-v h` on the command line will change to `-v HIGH`, which is ok.

Reply:
Oh, I see. I hadn't realised we had any non-UVM tests. Let's do a combination of our suggestions, so we keep `expand_uvm_verbosity_*` in `common_sim_cfg.hjson`, but use it in `run_opts` like `"+UVM_VERBOSITY={expand_uvm_verbosity_{verbosity}}` (a neat use of double-expansion!).

This means we can have a "verbosity" set in the Python code that doesn't have anything UVM-based (except possibly the choice of levels). Then everything UVM-specific ends up in the hjson files. I'll push up a patch doing this in a minute.